src/lib.rs

```diff
@@ -247,7 +248,7 @@ impl<'a> RobotFileParser<'a> {
         let client = Client::new();
         let request = client.get(self.url.clone())
             .header(UserAgent(USER_AGENT.to_owned()));
-        let mut res = match request.send() {
+        let res = match request.send() {
```

Comment:
I am not sure. 

https://doc.rust-lang.org/book/mutability.html

The book didn't mention anything like this.